client/src/components/Home/Sidebar/Profile/ProfileModal.js

```diff
@@ -5,19 +5,36 @@ import TextField from "@material-ui/core/TextField";
 import { withStyles } from "@material-ui/styles";
 
 import styles from '../../../../styles/ModalStyles';
+import StyledButton from '../../../../styles/StyledButton';
 
 class ProfileModal extends Component {
   state = { 
-    firstName: '',
-    lastName: ''
+    firstName: this.props.profile.firstName,
```

Comment:
Any reason why you do this instead of just using the props?

Reply:
The TextInputs are controlled components so their values are stored in the component state, but their initial values come from the props.

Reply:
Got it, makes sense